src/stories/Library/autosuggest-material/autosuggest-material.scss

```diff
@@ -30,9 +30,14 @@
       margin-bottom: 0;
     }
 
-    &:hover {
-      background-color: $c-global-secondary;
-      cursor: pointer;
+    @include breakpoint-m {
+      // Because the main autosuggest is a prominent feature we don't want the hover
+      // effect to take effect on phones, as iPhones have a known bug that requires
+      // users to double click in order to get past hover to register a click.
+      &:hover {
+        background-color: $c-global-secondary;
+        cursor: pointer;
```

Comment:
:+1: for commenting here.

Please add a source documenting/dealing with this issue as a part of the comment. When we claim that something as major as iPhones (Mobile Safari) has a known bug then adding a source adds some credence to our claim. It may also act as a way to determine if our workaround at some point can be removed.

In general we also have other elements with both hover states and click events (e.g. buttons). I wonder if these also suffer from the same problem.